lib/expression/callbacks/standard.ex

```diff
@@ -100,6 +114,77 @@ defmodule Expression.Callbacks.Standard do
     end
   end
 
+  @doc """
+  Calculates a new datetime based on a base date, a desired day and a desired time.
+
+  If no base date is supplied, it will fallback to current time.
+  """
+  @expression_doc doc:
+                    "Shifts a datetime to the next occurrence of a given day of the week and a set time.",
+                  expression: "next(\"monday\", \"10:00\", datetime)",
+                  context: %{
+                    "datetime" => ~U[2023-02-03 20:18:03Z],
+                    "number" => %{"from_addr" => "+5521987724322"}
+                  },
+                  fake_result: Map.merge(@datetime_struct, %{day: 6, hour: 10, minute: 0})
```

Comment:
I did it this way because `DateTime.new()` doesn't work here, I get a compilation error. I thought that having a fake result would still be pretty useful to get the automatically generated docs.